app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
+          setTimeout(function () { window.close(); }, 5000);
```

Comment:
I would even make it 10s, because the browser asks how to handle the `vmrc://` protocol and if the timeout exceeds, the window gets closed before the user can answer. The TTL of the ticket is 30 seconds, but because it was requested by the backend, we can't rely on it.